### PR TITLE
Fix k3s processing

### PR DESCRIPTION
**Pull request by dbason**

### CODE REVIEW

File: pkg/resources/preprocessor/workloads.go [hunk 1]
```diff
@@ -70,8 +70,13 @@ processors:
     log_statements:
     - context: log
       statements:
-      - set(attributes["COMM"], attributes["_COMM"])
-      - delete_matching_keys(attributes, "^_.*")
+      - merge_maps(cache, body, "upsert") where not IsString(body) and attributes["filename"] == nil
```
Comment (jaehnri) [inline pkg/resources/preprocessor/workloads.go:73]: How can the body not be a string? What else can it be?

**Review by jaehnri — APPROVED** [review]

File: pkg/resources/preprocessor/workloads.go (hunk 1 above)
Comment (jaehnri) [reply pkg/resources/preprocessor/workloads.go:73]: Ah, it can be a map, right?

**Review by alexandreLamarre — APPROVED** [review]
LGTM and validated with k3s 1.26.8

File: pkg/resources/preprocessor/workloads.go (hunk 1 above)
Comment (dbason) [reply pkg/resources/preprocessor/workloads.go:73]: In the case of journald logs it is a map.  That is specifically the case we want to handle.

**Pull request merged**